### PR TITLE
Add set/reset subcommand for game options

### PR DESCRIPTION
**Pull request by taahamahdi**

### CODE REVIEW

File: src/commands/game_options/multiguess.ts [hunk 1]
```diff
@@ -82,19 +82,42 @@ export default class MultiGuessCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.multiguess.help.interaction.description"
                     ),
```
Comment (Brainicism) [inline src/commands/game_options/multiguess.ts:117]: Wrong translations?

File: src/commands/game_options/ost.ts [hunk 2]
```diff
@@ -91,19 +91,42 @@ export default class OstCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.ost.help.description"
                     ),
```
Comment (Brainicism) [inline src/commands/game_options/ost.ts:126]: same here

File: src/commands/game_options/release.ts [hunk 3]
```diff
@@ -50,17 +50,42 @@ export default class ReleaseCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.release.help.interaction.description"
+                    ),
```
Comment (Brainicism) [inline src/commands/game_options/release.ts:85]: same here

File: src/commands/game_options/seek.ts [hunk 4]
```diff
@@ -91,17 +91,42 @@ export default class SeekCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.seek.help.description"
+                    ),
```
Comment (Brainicism) [inline src/commands/game_options/seek.ts:126]: same here

File: src/commands/game_options/shuffle.ts [hunk 5]
```diff
@@ -102,17 +103,42 @@ export default class ShuffleCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.shuffle.help.description"
+                    ),
```
Comment (Brainicism) [inline src/commands/game_options/shuffle.ts:138]: same here

File: src/commands/game_options/special.ts [hunk 6]
```diff
@@ -120,17 +120,42 @@ export default class SpecialCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.special.help.description"
+                    ),
```
Comment (Brainicism) [inline src/commands/game_options/special.ts:155]: same here

File: src/commands/game_options/subunits.ts [hunk 7]
```diff
@@ -92,20 +92,45 @@ export default class SubunitsCommand implements BaseCommand {
...
+                            ),
+                        },
+                    ],
+                },
+                {
+                    name: OptionAction.RESET,
+                    description: LocalizationManager.localizer.translate(
+                        LocaleType.EN,
+                        "command.subunits.help.description"
                     ),
```
Comment (Brainicism) [inline src/commands/game_options/subunits.ts:130]: same here and the rest

**Review by hoehoetvhss — DISMISSED** [review]

**Review by Brainicism — APPROVED** [review]

**Pull request merged**